networkx/algorithms/operators/binary.py

```diff
@@ -309,6 +309,30 @@ def compose(G, H):
     NodeView((0, 1, 2))
     >>> R.edges
     EdgeView([(0, 1), (0, 2), (1, 2)])
+
+    As indicated above, the attributes from H take precedent over attributes from G.
+    If you prefer another way of combining attributes, you can update them after the compose operation.
+    Consider the below graphs which have the nodes with the same labels, edge and node attributes:
+
+    >>> G = nx.Graph([('x', 'y', {'weight': 2.0}), ('x', 'z', {'weight': 1.0})])
+    >>> H = nx.Graph([('x', 'y', {'weight': 10.0}), ('y', 'z', {'weight': -1.0})])
+    >>> nx.set_node_attributes(G, {'x': "dark", 'y': "light", 'z':"neon"}, name='color')
+    >>> nx.set_node_attributes(H, {'x': "green", 'y': "orange", 'z':"yellow"}, name='color')
```

Comment:
This is entirely subjective, but I prefer using integer nodes for examples where the nodes themselves don't convey any special meaning, so I'd have a slight preference to replace `'x'`, `'y'`, and `'z'` with `0`, `1`, and `2`.

This is in no way a blocker though, so feel free to ignore!

Reply:
I thought that integer nodes might make the example less readable since there are many float numbers as weights too. Nonetheless, if you prefer integer nodes I can gladly change them. @rossbar

Reply:
That's a great point, and there's no objective reason to prefer one over the other - whichever way is totally fine!

Reply:
I don't think this needs to be on the same set of nodes and edges.
Can you make each graph have one node (and one edge) not in the other graph?
Then the example would show what happens when extra edges and missing edges arise.

```python 
>>> G = nx.Graph([(0, 1, {'weight': 3}, (1, 2, {'weight': 2})])
>>> H = nx.Graph([(1, 2, {'weight': 10}, (2, 3, {'weight': -1})])
```
I guess you'll need to use `.get('color', "")` and `.get('weight', 1)` so it can handle the missing cases...

Reply:
I modified the example such that it also prints the updated attributes of unique node/edge. Is it clear enough? Thanks.